packages/manager/apps/dedicated/client/app/app.js

```diff
@@ -240,4 +241,10 @@ angular
         });
       });
   })
-  .constant('UNIVERSE', 'DEDICATED');
+  .constant('UNIVERSE', 'DEDICATED')
+  .config(/* @ngInject */($httpProvider) => {
+    $httpProvider.interceptors.push('OvhNgRequestTaggerInterceptor');
```

Comment:
This is already done by `@ovh-ux/manager-core`

Reply:
https://github.com/ovh/manager/blob/master/packages/manager/modules/core/src/index.js#L186